src/Tests/Cache/PhpRedisUnitTest.php

```diff
@@ -45,7 +44,11 @@ public function containerBuild(ContainerBuilder $container) {
    *   A new PhpRedis cache backend.
    */
   protected function createCacheBackend($bin) {
-    $cache = \Drupal::service('cache.backend.redis')->get($bin);
+    $cache = new PhpRedis(
```

Comment:
I was forced by my environment to force the backend class this way, else it would sometime arbitrarily use the other in the test, not sure why. This way we're sure it works fine.